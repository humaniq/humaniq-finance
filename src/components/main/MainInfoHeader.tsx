import React, { ReactNode } from "react";

export interface MainInfoHeaderProps {
  backgroundColor?: string;
  children: ReactNode;
}

export const MainInfoHeader = ({
  backgroundColor = "#001833",
  children,
}: MainInfoHeaderProps) => {
  return <div style={{ backgroundColor, width: "100%" }}>{children}</div>;
};
