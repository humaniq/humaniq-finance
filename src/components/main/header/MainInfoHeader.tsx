import React, { ReactNode } from "react";
import "./MainInfoHeader.style.sass";

export interface MainInfoHeaderProps {
  backgroundColor?: string;
  style?: any;
  className?: string;
  children: ReactNode;
}

/**
 * Main info header component
 *
 * @param children
 * @param style
 * @param className
 * @constructor
 */
export const MainInfoHeader: React.FC<MainInfoHeaderProps> = ({
  children,
  style = {},
  className,
}) => {
  return (
    <div style={style} className={`main-info-container ${className}`}>
      {children}
    </div>
  );
};
