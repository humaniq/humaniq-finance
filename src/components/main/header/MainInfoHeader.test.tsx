import * as renderer from 'react-test-renderer';
import {MainInfoHeader} from "./MainInfoHeader"

describe("MainInfoHeader component", () => {
  it("should natch snapshot", () => {
    const { toJSON } = renderer.create(<MainInfoHeader><div></div></MainInfoHeader>);
    expect(toJSON()).toMatchSnapshot();
  });
});
