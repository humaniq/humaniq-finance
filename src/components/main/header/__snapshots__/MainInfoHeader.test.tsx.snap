// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainInfoHeader component should natch snapshot 1`] = `
<div
  className="main-info-container undefined"
  style={Object {}}
>
  <div />
</div>
`;
